State that additional properties are ignored (#284)
graphql/graphql-over-http: spec/GraphQLOverHTTP.md

All other property names are reserved for future expansion; if implementors need
to add additional information to a request they MUST do so via other means, the
RECOMMENDED approach is to add an implementor-scoped entry to the {extensions}

> object. Servers receiving a request with additional properties MUST ignore the
> additional properties when processing the request.

Reviewer: ```suggestion
object.

Servers receiving a request with additional properties MUST ignore properties
they do not understand.
```

I've changed this to "properties they do not understand" to allow for them to implement properties mentioned in extensions, proposals, etc. The "MUST do so via other means" above already precludes them adding their own (vendor) properties.
